service/src/test/java/bio/terra/workspace/service/workspace/GcpCloudContextConnectedTest.java

```diff
@@ -331,114 +306,4 @@ public void cloneGcpWorkspace() throws Exception {
             .listFoldersInWorkspace(destinationWorkspace.getWorkspaceId())
             .contains(sourceFolder));
   }
-
-  @Test
-  @Disabled("PF-2259 - this test needs rewriting")
```

Comment:
This test was disabled, and it was the only remaining places calling `LaunchCreateCloudContextFlightStep` and `AwaitCreateCloudContextFlightStep`, so I'm deleting it.